data/locale.yml

```diff
@@ -30,10 +30,15 @@ keyboard_modal:
 docs:
   title: Electron Documentation
   description: See all of the <a href="/docs/all">docs on one page</a> or check out the <a href="/docs/faq">FAQ</a>.
-  guides: Guides
+  guides: "Guides: Get Started with Electron"
```

Comment:
💭 We use a lot of present-tense verbs in our titles and headings; what do you think about "Getting Started" instead?